src/content/community/releases/_index.en.md

```diff
@@ -67,6 +67,16 @@ weight = 40
   In some scenarios however, an invalid directory is created before OVN is upgraded, which prevents OVN from starting up correctly.
   This will be fixed fully in v0.16.1.
 
+## v0.14.7 (October 17, 2023)
```

Comment:
We put them in chronological order so this should be at the top.

Reply:
Done.